sender/rabbitmq_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+ * skogul, rabbitmq producer/sender test
+ *
+ * Copyright (c) 2023 Telenor Norge AS
+ * Author(s):
+ *  - Kamil Oracz <kamil.oracz@telenor.no>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301  USA
+ */
+
+package sender
+
+import (
+	"testing"
+	"time"
+
+	"github.com/telenornms/skogul"
+)
+
+func createContainer() *skogul.Container {
+	meta := make(map[string]interface{})
+	meta["foo"] = "bar"
+	data := make(map[string]interface{})
+	data["baz"] = "qux"
+
+	metric := skogul.Metric{
+		Time:     &time.Time{},
+		Metadata: meta,
+		Data:     data,
+	}
+	metrics := make([]*skogul.Metric, 0)
+	metrics = append(metrics, &metric)
+
+	return &skogul.Container{
+		Metrics: metrics,
+	}
+}
+
+func TestRabbitmq(t *testing.T) {
+	data := createContainer()
```

Comment:
You can skip this test in CI because there is no rabbitmq-listener to send to using `if testing.Short()` like this: https://github.com/telenornms/skogul/blob/primary/sender/sql_test.go#L147

That'll make the CI checks ✅